python/preemo/worker/_worker_client.py

```diff
@@ -91,6 +97,74 @@ def __call__(self, params: Optional[bytes] = None) -> Optional[Result]:
 
 # TODO(adrian@preemo.io, 05/11/2023): add logging
 class WorkerClient:
+    @staticmethod
+    def _ensure_value_is_non_negative(
+        *, name: str, value: Optional[Union[int, float]]
+    ) -> None:
+        if value is not None and value < 0:
+            raise Exception(f"{name} must not be negative")
```

Comment:
what does this have to do with a worker client

Reply:
nothing. I can move it